kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/modularrouters/shapeless.js

```diff
@@ -22,7 +29,7 @@ onEvent('recipes', (event) => {
                 'tconstruct:sky_slime_crystal',
                 'tconstruct:sky_slime_crystal'
             ],
-            id: 'modularrouters:stack_upgrade'
+            id: `'modularrouters:stack_upgrade'`
```

Comment:
Typo I think 🤔

Reply:
Well that's a weird place for that to happne >.>